Django_Blog/blog/models.py

```diff
@@ -1,3 +1,29 @@
 from django.db import models
 
 # Create your models here.
+class Author(models.Model):
+    name = models.CharField(max_length=100,null=False)
+    about_author = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+class Blog(models.Model):
+    author = models.ForeignKey(Author, on_delete=models.CASCADE)
+    title = models.CharField(max_length=100)
+    blog_pub_date = models.DateTimeField(auto_now_add=True)
```

Comment:
Create a BaseModel that have two fields `created_at` and `updated_at` and inherits this in each model you created so you don't have to write like `blog_pub_date` and `comment_date`.